src/helm/benchmark/scenarios/bhasa_scenario.py

```diff
@@ -0,0 +1,1908 @@
+#   D. Linguistic Diagnostics
+
+# A. Natural Language Understanding
+#   1. Question Answering
+#   2. Sentiment Analysis
+#   3. Toxicity Detection/Classification
+
+# 1. Question Answering
+# 1.1 Indonesian: TyDiQA
+class TyDiQAScenario(Scenario):
+    """
+    TyDiQA is is an open-book question answering scenario for 11 typologically-diverse languages.
+    The questions are written by people who want to know the answer, but do not know the answer yet,
+    and the data is collected directly in each language without the use of translation.
+
+    This scenario only uses the Indonesian subset of the data, and uses the Gold Passage (GoldP) task,
+    which requires the tested system to extract a span from the given passage to answer a given question.
+    There are no unanswerable questions.
+
+    The models are prompted using the following format:
+
+        Anda akan diberikan sebuah paragraf dan sebuah pertanyaan. Jawablah pertanyaannya dengan mengekstrak jawaban dari paragraf tersebut.
+
+        Paragraf: <text>
+        Pertanyaan: <question>
+        Jawaban: <answer>
+
+        ...
+
+        Paragraf: <text>
+        Pertanyaan: <question>
+        Jawaban:
+
+
+    Target completion:
+        <answer>
+
+    @article{clark-etal-2020-tydi,
+        title = "{T}y{D}i {QA}: A Benchmark for Information-Seeking Question Answering in Typologically Diverse Languages",
+        author = "Clark, Jonathan H.  and
+        Choi, Eunsol  and
+        Collins, Michael  and
+        Garrette, Dan  and
+        Kwiatkowski, Tom  and
+        Nikolaev, Vitaly  and
+        Palomaki, Jennimaria",
+        editor = "Johnson, Mark  and
+        Roark, Brian  and
+        Nenkova, Ani",
+        journal = "Transactions of the Association for Computational Linguistics",
+        volume = "8",
+        year = "2020",
+        address = "Cambridge, MA",
+        publisher = "MIT Press",
+        url = "https://aclanthology.org/2020.tacl-1.30",
+        doi = "10.1162/tacl_a_00317",
+        pages = "454--470",
+    }
+    """
+
+    name = "tydiqa"
+    description = "Indonesian Open-book Question Answering task"
+    tags = ["question_answering"]
+
+    def __init__(self):
+        super().__init__()
+        self.splits = {
+            "train": TRAIN_SPLIT,
+            "validation": TEST_SPLIT
+        }
+
+    def get_instances(self, output_path) -> List[Instance]:
+        dataset = datasets.load_dataset("khalidalt/tydiqa-goldp", "indonesian")
+
+        outputs = []
+        for split in self.splits.keys():
+            df = dataset[split].to_pandas()
+
+            if split == "train":
+                # Select only bottom 20th percentile by length for in-context examples as examples are very long
+                data = df[df["passage_text"].apply(len) < df["passage_text"].apply(len).quantile(.2)]
+            else:
+                # Sample 100 examples for test
+                data = df.sample(n=100, random_state=5018)
```

Comment:
100 seems a bit small, could we do 1k or 10k? Or does this cause memory issues.

Reply:
You're right, we initially used a small number for QA and summarization because the texts were very long and we had limited resources, but ideally a larger number would be better for more statistical power.

I initially thought we might want to keep it as close to what we used in the BHASA paper as possible, but I suppose we could just apply the improvement now too.

The maximum we can do here would be 565 (the size of the validation set).

Reply:
OK - also note that `helm-run` also does downsampling if you pass in a flag like `--max-eval-instances 100`.

Reply:
Hi @weiqipedia, to clarify, should we increase the sample size to the maximum of the test set for TyDiQA only? Or also do the same for the other scenarios (such as XQuAD and IndicQA?)

Reply:
We can increase the sampling to 1000 for XQuAD and IndicQA, and no need to sample for TyDiQA since we are just taking the entire validation set